addons/lunch/static/src/components/lunch_dashboard.js

```diff
@@ -0,0 +1,179 @@
+/** @odoo-module */
+
+import { useBus, useService } from "@web/core/utils/hooks";
+import { Many2XAutocomplete } from "@web/views/fields/relational_utils";
+
+const { Component, useState, onWillStart, markup, xml } = owl;
+
+export class LunchCurrency extends Component {
+    get amount() {
+        return parseFloat(this.props.amount).toFixed(2);
+    }
+}
+LunchCurrency.template = 'lunch.LunchCurrency';
+LunchCurrency.props = ["currency", "amount"];
+
+export class LunchOrderLine extends Component {
+    setup() {
+        super.setup();
+        this.orm = useService('orm');
+        this.state = useState({ mobileOpen: false });
+    }
+
+    get line() {
+        return this.props.line;
+    }
+
+    get canEdit() {
+        return !['sent', 'confirmed'].includes(this.line.raw_state);
+    }
+
+    get badgeClass() {
+        const mapping = {'new': 'bg-warning', 'confirmed': 'bg-success', 'sent': 'bg-info', 'ordered': 'bg-danger'};
+        return mapping[this.line.raw_state];
+    }
+
+    get hasToppings() {
+        return this.line.toppings.length !== 0;
+    }
+
+    async updateQuantity(increment) {
+        await this.orm.call('lunch.order', 'update_quantity', [
+            this.props.line.id,
+            increment
+        ]);
+
+        await this.props.onUpdateQuantity();
+    }
+}
+LunchOrderLine.template = 'lunch.LunchOrderLine';
+LunchOrderLine.props = ["line", "currency", "onUpdateQuantity", "openOrderLine"];
+LunchOrderLine.components = {
+    LunchCurrency,
+};
+
+export class LunchAlert extends Component {
+    get message() {
+        return markup(this.props.message);
```

Comment:
Hi @odoo/rd-security :wave: 

Could you have a look at this PR please? :-)

`markup()` is used here to render the content of the alert's message (which is an html field), before it was done with a t-out[1]. The alerts are coming through this controller here[2]


[1] : https://github.com/odoo/odoo/blob/3bb0aaaa0833e3d72fbb07e47e42441e293babb7/addons/lunch/static/src/xml/lunch_templates.xml#L6
[2] : https://github.com/odoo/odoo/blob/8295bd1ba0dde8688defbb2fdf6e287ef2da7273/addons/lunch/controllers/main.py#L118

Reply:
Why is the `message` an HTML field btw? Any lunch user can create and modify lunch alert messages, hence inject html (thankfully it's sanitized but still, not sure if necessary)